tests/packagedcode/data/chef/dependencies/metadata.rb.expected

```diff
@@ -35,16 +35,16 @@
   "manifest_path": null,
   "dependencies": [
     {
-      "purl": "pkg:chef/mingw",
-      "requirement": ">= 1.1",
+      "purl": "pkg:chef/seven_zip",
+      "requirement": null,
```

Comment:
I am surprised as why we do not get stable ordering there: is this because we are not using an OrderedDict

Reply:
Leave it because you have to delete the test for py2